solara/website/pages/documentation/getting_started/content/07-deploying/10-self-hosted.md

```diff
@@ -65,7 +65,7 @@ A typical command would be:
 $ SOLARA_APP=sol.py gunicorn --workers 4 --threads=20 -b 0.0.0.0:8765 solara.server.flask:app
 ```
 
-Note that we need at least 1 thread per user due to the use of a websocket.
+Note that we need at least 1 thread per user due to the use of a websocket. *Make sure you understand the implications of using multiple workers by reading [about solara server](https://solara.dev/documentation/advanced/understanding/solara-server)*
```

Comment:
Should these instead link to the specific section to make it clearer the documentation we link to is directly related?
```suggestion
Note that we need at least 1 thread per user due to the use of a websocket. *Make sure you understand the implications of using multiple workers by reading [about handling multiple workers with solara server](https://solara.dev/documentation/advanced/understanding/solara-server#handling-multiple-workers)*
```